README.md

```diff
@@ -396,29 +396,69 @@ MIT
 
 ## Contributing
 
-### Tests
-
-Tests are written using the HubSpot API sandbox. As such, resources get added
-and deleted all the time.
-
-Don't hard-code record values when fetching / updating / deleting (because those
-tend to disappear). Instead, fetch first an array of available records, then use
-of the returned ID in your test.
-
-### Contributors
-
-- Brian Falk @brainflake
-- Tim Atkinson @timisbusy
-- Tejas Manohar @tejasmanohar
-- Krispin Schulz @kr1sp1n
-- Filipe Ferreira @iTsFILIPOficial
-- Sam Levan @samlevan
-- Paul Cothenet @pcothenet
-- Nick Mason @masonator
-- Mikael Puittinen @mpuittinen
-- @davidmfoley
-- @jayprakash1
-- @alexatdivvy
-- @forstermatth
-- @amit777
-- Stas Nikiforov @stasnikiforov
+Before submitting a pull request, please make sure the following is done:
+
+1. Fork the repository and create your branch from master.
+1. Run `npm install` in the repository root.
+1. If you’ve fixed a bug or added code that should be tested, add tests!
+1. Ensure the test suite passes `npm test`. Tip:
+   `npm run mocha -- --grep "test name"` is helpful in development.
+1. Format your code with prettier and eslint `npm run lint`.
+
+## Development Workflow
+
+After cloning this repo, run `npm install` to fetch dependencies. Then run the
+test suite with `npm test`. From master, this should be green. Our tests mock
+out requests to HubSpot's api using [nock]. We then recommend running the test
+suite without mocking api requests with `NOCK_OFF=true npm run mocha`. This
+should fail as you'll need some [environment variables] for real requests to
+pass.
+
+[nock]: https://github.com/nock/nock
+[environment variables]: https://github.com/motdotla/dotenv
+
+If you haven't already, create a [developer account] on hubspot. You'll want to
+[create an app] and a [test account] as well. Then, create a new file, `.env` in
+the root of the repo. Inside you'll need to add an [app id], a HubSpot user id,
+an [api key] and an [oauth access token];
```

Comment:
> an [api key] and an [oauth access token]

Should this be an "or"? I believe only on approach is necessary.

> an [api key] ~and~ or an [oauth access token]

Reply:
Unfortunately not. For the test suite to pass you need both as we have tests that use both methods to authenticate.